app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
```

Comment:
1. nói cho chị bjk sự khác nhau của find, dinf_by và find_by! nhes

2. owe đây dùng `find_by` và nhớ code xử lý trương hợp user bị nil

Reply:
nếu khi ta sử dụng Find thì khi không tìm thấy record nào, nó sẽ bắn ra một exception, exception này khiến server bị lỗi và gián đoạn, còn khi dùng Find_by thì khi k tìm thấy record nào thì nó return về nill, Find_by! thì e chưa rõ lắm nhưng em nghĩ thì khi nó trả về nil thì nó gắn giá trị vào biến đang tìm là nil luôn ạ

Reply:
So với những gì được overview buổi hôm trước thì mới trả lời được 1/3 ý thôi nè

Reply:
```suggestion
    @user = User.find_by id: params[:id]
```
chuyển sang dùng find_by và xử lý thêm trường hợp tìm nhưng bị nil nữa e nha